src/main/java/net/minecraftforge/event/dimension/DimensionRegisterEvent.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.event.dimension;
+
+import net.minecraftforge.dimension.DynamicDimensionManager;
+import net.minecraftforge.eventbus.api.Event;
+
+/**
+ * Register your custom dimension definitions when you receive this event.
```

Comment:
no need for so much space in the docs, plus wrap that huge line on multiple ones instead.